src/praisonai/praisonai/gateway/home_channels.py

```
@@ -132,13 +132,51 @@ class DeliveryResolver(DeliveryResolverProtocol):
+        Returns a concrete ``DeliveryTarget`` when the directory knows the
+        alias, otherwise ``None`` so the caller can fall through to the
+        existing home-channel resolution.
+        """
+        if self._directory is None:
+            return None
+        resolve_alias = getattr(self._directory, "resolve_alias", None)
+        if not callable(resolve_alias):
+            return None
+        try:
+            resolved = resolve_alias(token)
+        except Exception as e:  # pragma: no cover - defensive
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)